recipe/meta.yaml

```diff
@@ -40,11 +40,17 @@ test:
     - quantecon.markov
     - quantecon.random
     - quantecon.util
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/QuantEcon/QuantEcon.py
+  home: https://quantecon.org/quantecon-py/
+  license_url: https://github.com/QuantEcon/QuantEcon.py/blob/v{{ version }}/LICENSE
```

Comment:
I'm not entirely sure if it accepts an URL. You might be better to simply use `LICENSE`. As per https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#license-file, you can reference a file relative to the source directory (the unpacked archive in this case).

Reply:
Got it! I'll make the change. Thanks!